google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/Connection.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.bigquery;
+
+import com.google.api.core.BetaApi;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * A Connection is a session between a Java application and BigQuery. SQL statements are executed
+ * and results are returned within the context of a connection.
+ */
+public interface Connection {
+
+  /** Sends a query cancel request. This call will return immediately */
+  @BetaApi
+  Boolean close() throws BigQuerySQLException;
```

Comment:
Any particular reason for `Boolean` over boolean? In other words, can `null` be returned here?

Reply:
Done